File: java/src/main/java/de/sven_torben/serialization_benchmark/TestResult.java
package de.sven_torben.serialization_benchmark;

class TestResult<T> {
	
	private long time;

	public long getTime()

	{
		return time;
	}

	public void setTime(final long time) {
		this.time = time;
	}

	private T data;

	public T getData() {
		return data;
	}

	public void setData(final T data) {
		this.data = data;
	}
}

File: net/TestResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace De.Sven_Torben.Serialization_Benchmark
{
    class TestResult<T>
    {
        public long Time { get; set; }
        public T Data { get; set; }
    }
}
